src/deploy/executor.rs

```diff
@@ -556,19 +567,25 @@ where
 			}
 		}
 
-		if file.template_or_default() {
-			if !self.options.dry_run {
-				let content = match std::fs::read(&file_source_path) {
-					Ok(content) => content,
-					Err(err) => {
-						log::info!("[{}] Failed to read source content", file.path.display());
-						builder.add_item(file_deploy_path, file, err.into());
-						return Ok(());
-					}
-				};
+		if file.is_template() {
+			let content = match std::fs::read_to_string(&file_source_path) {
+				Ok(content) => content,
+				Err(err) => {
+					log::info!("[{}] Failed to read source content", file.path.display());
```

Comment:
Why dont throw a real error (with added context using eyre) here?

Reply:
We will have to discuss this. If we throw an error here the whole deploying process will be aborted. This could be one way of handling this. The other way could be to inform the user with an error log and continue with the deployment.

Reply:
At least we should use `log::error!`. I think it's fine to not abort the whole process, since other files might be fine

Reply:
You are absolutely right with that it should be `log::error`.